File: boj/1110.py
# https://www.acmicpc.net/problem/1110
def operation(num):
    return num % 10 * 10 + (num // 10 + num % 10) % 10


original_num = int(input())
num = operation(original_num)
ctr = 1
while num != original_num:
    num = operation(num)
    ctr += 1

print(ctr)

File: boj/1110.js
// https://www.acmicpc.net/problem/1110
"use strict";

const num = Number(require("fs").readFileSync("/dev/stdin"));
let num2 = num;
let cnt = 0;
do {
  // return num % 10 * 10 + (num // 10 + num % 10) % 10
  num2 = (num2 % 10) * 10 + ((Math.floor(num2 / 10) + (num2 % 10)) % 10);
  cnt++;
} while (num2 != num);

console.log(cnt);
